docs/cli.md

```
@@ -0,0 +1,906 @@
+| `init` | Print agent init prompt |
+
+#### hack agent init
+
+Usage: `hack agent init [options]`
+
+Options:
+
+| Flag | Type | Default | Description |
+| --- | --- | --- | --- |
+| `-p`, `--path <dir>` | string | - | Run against a repo path (overrides cwd search) |
+| `-c`, `--client <cursor|claude|codex|print>` | string | - | Open init prompt in an agent client (or print) |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 4; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)